numpy/random/tests/test_randomstate.py

```diff
@@ -1320,134 +1319,127 @@ class TestBroadcast:
     seed = 123456789
 
     def set_seed(self):
-        random.seed(self.seed)
+        return random.RandomState(self.seed)
```

Comment:
The name of this method is no longer appropriate. Feel free to inline `random.RandomState(self.seed)` everywhere in this class (or rename this method if it's easier).